tests/test_plot.py

```diff
@@ -326,7 +349,8 @@ def test_plot_consistency():
     labels4 = np.random.uniform(0, 1, (500,))
     labels_datasets = [labels1, labels2, labels3, labels4]
 
-    dataset_ids = ["achilles", "buddy", "buddy", "achilles"]
+    dataset_ids = ["achilles", "buddy", "cicero", "gatsby"]
+    dataset_ids_runs = ["achilles", "buddy", "buddy", "achilles"]
```

Comment:
Still TODO: Would be good to double check that the previous format with repeated datasets, `["achilles", "buddy", "buddy", "achilles"]` does indeed not make sense when plotting the consistency matrix --- if both this behavior plus #27 should be supported, we need additional internal changes to the plot (e.g., how numbers should be averaged during plotting).